css/style.css

```diff
@@ -0,0 +1,92 @@
+
+/*#fadeImage{
+  background: white;
+}*/
+
+
+#container{
+  display: none;
+  max-width: 960px;
+  margin: auto;
+}
+
+body{
+  background: url("space4.gif");
+}
+
+#page-header{
+  border:0;
+  box-sizing: border-box;
+  clear: both;
+  display: block;
+  float: left;
+  margin: auto;
+  padding: 2% 8% 2% 8%;
+  width: 100%;
+}
+
+.clearfix{
```

Comment:
Clearfix is generally understood to take a very specific action, which is actually more than just clearing the floats. The CSS for this class should be:
```
.clearfix:after {
  content: "";
  display: table;
  clear: both;
}
```